File: fjt.api/models/RFQAssyQuantity.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const RFQAssyQuantity = sequelize.define('RFQAssyQuantity', {
        id: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER
        },
        rfqAssyID: {
            allowNull: false,
            type: Sequelize.INTEGER
        },
        rfqPriceGroupId: {
            allowNull: true,
            type: Sequelize.INTEGER
        },
        rfqPriceGroupDetailId: {
            allowNull: true,
            type: Sequelize.INTEGER
        },
        requestQty: {
            allowNull: false,
            type: Sequelize.INTEGER
        },
        quantityType: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        materialTotal: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        materialHandling: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        materialScrapPercentage: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        materialScrap: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        materialCarryingCostPercentage: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        materialCarryingCost: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        excessQtyTotal: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        excessTotalDollar: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        leadCostTotal: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        attritionRateTotal: {
            type: Sequelize.DECIMAL(10, 0),
            allowNull: true
        },
        copyQtyId: {
            allowNull: false,
            type: Sequelize.INTEGER
        },
        isDeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        createdAt: {
            type: Sequelize.DATE,
            allowNull: true
        },
        updatedAt: {
            type: Sequelize.DATE,
            allowNull: true
        },
        deletedAt: {
            type: Sequelize.DATE,
            allowNull: true
        },
        createByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        updateByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        deleteByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        }
    }, {
        paranoid: true,
        tableName: 'rfq_assy_quantity'
    });

    RFQAssyQuantity.associate = (models) => {
        RFQAssyQuantity.hasMany(models.RFQAssyQuantityTurnTime, {
            as: 'rfqAssyQtyTurnTime',
            foreignKey: 'rfqAssyQtyID'
        });

        RFQAssyQuantity.belongsTo(models.RFQAssemblies, {
            as: 'rfqAssemblies',
            foreignKey: 'rfqAssyID'
        });

        RFQAssyQuantity.hasMany(models.RFQConsolidatedMFGPNLineItem, {
            as: 'rfqConsolidatedMFGPNLinteItem',
            foreignKey: 'qtyID'
        });

        RFQAssyQuantity.hasMany(models.RFQAssyQuotations, {
            as: 'rfqAssyQuotations',
            foreignKey: 'rfqAssyQtyID'
        });

        RFQAssyQuantity.hasMany(models.RFQConsolidatedMFGPNLineItemQuantity, {
            as: 'rfqConsolidatedMFGPNLineItemQuantity',
            foreignKey: 'qtyID'
        });
        RFQAssyQuantity.belongsTo(models.RFQPriceGroup, {
            as: 'rfqPriceGroup',
            foreignKey: 'rfqPriceGroupId'
        });
        RFQAssyQuantity.belongsTo(models.RFQPriceGroupDetail, {
            as: 'rfqPriceGroupDetail',
            foreignKey: 'rfqPriceGroupDetailId'
        });
    };

    return RFQAssyQuantity;
};

File: fjt.pricingservice/MySqlDBModel/rfq_assy_quantity.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;

namespace fjt.pricingservice.MySqlDBModel
{
    [Table("rfq_assy_quantity")]
    public partial class rfq_assy_quantity
    {
        public rfq_assy_quantity()
        {
            rfq_assy_quantity_turn_time = new HashSet<rfq_assy_quantity_turn_time>();
            rfq_consolidate_mfgpn_lineitem_quantity = new HashSet<rfq_consolidate_mfgpn_lineitem_quantity>();
            rfq_assy_quantity_price_selection_setting= new HashSet<rfq_assy_quantity_price_selection_setting>();
        }
        [Key]
        public int id { get; set; }
        public int rfqAssyID { get; set; }
        public int requestQty { get; set; }
        public decimal? materialTotal { get; set; }
        public decimal? materialHandling { get; set; }
        public decimal? materialScrapPercentage { get; set; }
        public decimal? materialScrap { get; set; }
        public decimal? materialCarryingCostPercentage { get; set; }
        public decimal? materialCarryingCost { get; set; }
        public decimal? excessQtyTotal { get; set; }
        public decimal? excessTotalDollar { get; set; }
        public decimal? leadCostTotal { get; set; }
        public decimal? attritionRateTotal { get; set; }
        public int? copyQtyId { get; set; }
        public bool isDeleted { get; set; }
        [StringLength(255)]
        public string createdBy { get; set; }
        public DateTime createdAt { get; set; }
        [StringLength(255)]
        public string updatedBy { get; set; }
        public DateTime? updatedAt { get; set; }
        [StringLength(255)]
        public string deletedBy { get; set; }
        public DateTime? deletedAt { get; set; }
        public virtual rfq_assemblies rfq_assemblies { get; set; }
        public virtual ICollection<rfq_assy_quantity_turn_time> rfq_assy_quantity_turn_time { get; set; }
        public virtual ICollection<rfq_consolidate_mfgpn_lineitem_quantity> rfq_consolidate_mfgpn_lineitem_quantity { get; set; }
        public virtual ICollection<rfq_assy_quantity_price_selection_setting> rfq_assy_quantity_price_selection_setting { get; set; }
    }
}
